portal-impl/test/integration/com/liferay/portal/service/OrganizationLocalServiceTest.java

```diff
@@ -300,6 +300,32 @@ public void testGetNoAssetOrganizations() throws Exception {
 	}
 
 	@Test
+	public void testGetOrganizationsAndUsers() throws Exception {
```

Comment:
This test is fine, but since this code is tricky I think we should add more tests covering more test cases. For example (but please don't be limited to this):
- Having no users
- Having no orgs
- Having neither of them
- All of the above for the root folder
- All of the above for the search methods